### DISCUSSION TITLE
sqlte

### DISCUSSION THREAD

**Discussion by emanu3la2** [pawelsalawa/sqlitestudio#5104] [Q&A]
### Details
Here you describe your problem details, or proposal for enhancement.

### Steps to reproduce
This applies only to bug reports. If this is a feature request, delete this section.

### Operating system
This applies only to bug reports. If this is a feature request, delete this section.

### SQLiteStudio version
This applies only to bug reports. If this is a feature request, delete this section.

**Comment by emanu3la2** [comment]
i'm just trying to add a database on sqlite, but it doesn't appear, idk why, I really don't understand, can u help me?

**Comment by pawelsalawa** [comment]
So... how are you trying to do it? What steps do you do?
